parquet-avro/src/test/java/org/apache/parquet/avro/TestArrayCompatibility.java

```diff
@@ -653,8 +654,8 @@ public void testAvroCompatRequiredGroupInListWithSchema() throws Exception {
     Path test = writeDirect(
         "message AvroCompatRequiredGroupInListWithSchema {" +
             "  optional group locations (LIST) {" +
-            "    repeated group array {" +
-            "      optional group element {" +
+            "    repeated group bag {" +
+            "      optional group array_element {" +
```

Comment:
@rdblue Just noticed the `testHiveCompatOptionalGroupInList` test case, which is very similar to this one.  I'm a little bit confused by the purpose of this test case now, especially the `RequiredGroupInListWithSchema` part of this test case name: there's no required group appeared in the testing schema.